src/imp/security_framework.rs

```diff
@@ -14,9 +15,16 @@ use self::tempdir::TempDir;
 use std::fmt;
 use std::io;
 use std::error;
+use std::sync::Mutex;
+use std::sync::atomic::{AtomicBool, Ordering};
 
 use Protocol;
 
+lazy_static! {
+    static ref AT_EXIT_SET: AtomicBool = AtomicBool::new(false);
+    static ref TEMP_DIRS: Mutex<Vec<TempDir>> = Mutex::new(vec![]);
```

Comment:
Can we treat this as a map from the raw DER data to the directory? That way we don't end up with tons of directories when the same identity is imported multiple times.